samples/nrf9160/serial_lte_modem/README.rst

```diff
@@ -141,6 +141,22 @@ The following proprietary GPS AT commands are used in this sample:
 
 * AT#XGPS=<op>[,<mask>]
 
+MQTT AT Commands
+****************
+
+The following proprietary MQTT AT commands are used in this sample:
+
+* AT#XMQTTCON=<op>[,<url>,<port>[,<sec_tag>]]
+* AT#XMQTTPUB=<topic>,<datatype>,<msg>,<qos>,<retain>
+* AT#XMQTTSUB=<topic>,<op>[,<qos>]
+
+The following unsolicited notification reports the MQTT events.
```

Comment:
This is hard to understand. Maybe:
```suggestion
The following unsolicited notification indicates that an MQTT event occurred:
```

Reply:
Thank you. I've modified the description as:

The following unsolicited notification indicates that an MQTT event occurred:

* #XMQTTEVT=<type>,<result>